library/std/src/os/mod.rs

```diff
@@ -161,3 +162,22 @@ pub mod fd;
 
 #[cfg(any(target_os = "linux", target_os = "android", doc))]
 mod net;
+
+/// Breakpoint that is passed to the debugger as the first chance exception
+/// if the debugger is attached, and is skipped over otherwise.
+///
+/// Returns `1` if the breakpoint was handled by the debugger,
+/// and `0` is returned if the breakpoint was handled by the OS
+/// exception dispatcher.
+#[cfg(not(target_os = "windows"))]
+#[unstable(feature = "dbg_breakpoint", issue = "none")]
+pub fn dbg_breakpoint() -> i32 {
+    // Implemented for Windows only at the moment.
+    0
+}
+
+#[cfg(target_os = "windows")]
+#[unstable(feature = "dbg_breakpoint", issue = "none")]
+pub fn dbg_breakpoint() -> i32 {
+    crate::os::windows::dbg::dbg_breakpoint()
+}
```

Comment:
According to this documentation, 0 is an incorrect return for this code because no breakpoint was hit if `dbg_breakpoint()` was called on other OS. That means **nothing** was handled by the OS exception dispatcher.

But I do not think we should add something returning "magic numbers" to the Rust standard library.

Reply:
Understood. Can remove the return value w/o any loss to the functionality if you see this as a good solution.